radar/languages-and-frameworks/spring-boot.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Spring Boot"
+ring: adopt
+quadrant: "languages-and-frameworks"
+tags: [java, backend]
+---
+
+Spring Boot is an open source Java-based framework used to create a microservices. It is developed by Pivotal Team and is used to build stand-alone and production ready spring applications.
+
+The Spring Cloud ecosystem also gives you a lot of extension points for developing, deploying and running cloud applications.
+
+It's based on the rock-solid Spring framework and provides excellent documentation.
+
+[Spring Boot](https://spring.io/projects/spring-boot)
+
+### Key Features
+
+- **Great community support**: the large community behind spring boot and Pivotal itself allow for ready-to-use integrations for most commonly used technologies.
+
+- **Spring Cloud**: provides tools for developers to quickly build some of the common patterns in distributed systems. Coordination of distributed systems leads to boiler plate patterns, and using Spring Cloud developers can quickly stand up services and applications that implement those patterns.
+[Spring Cloud](https://spring.io/projects/spring-cloud)
+
+- **Spring Data**: the mission is to provide a familiar and consistent, Spring-based programming model for data access while still retaining the special traits of the underlying data store. It makes it easy to use data access technologies, relational and non-relational databases, map-reduce frameworks, and cloud-based data services. 
+[Spring Data](https://spring.io/projects/spring-data)
```

Comment:
To decide if we want to adopt these as well, these other technologies should be categorized in separate, distinct entries.

Reply:
I'm agree, it's better to move it as distinct entries